src/Lachain.Console/Application.cs

```diff
@@ -92,15 +99,33 @@ public void Start(RunOptions options)
             commandManager.Start(wallet.EcdsaKeyPair);
             rpcManager.Start();
 
+            consensusManager.Start((long)blockManager.GetHeight() + 1);
+            validatorStatusManager.Start(false);
+
+            Thread.Sleep(10000);
+
+            if (blockManager.GetHeight() == 0)
+            {
+                blockSynchronizer.StartFastSync();    
+            }
+            
+            // consensusManager.Start((long)blockManager.GetHeight() + 1);
+            // validatorStatusManager.Start(false);
+
```

Comment:
```suggestion
            if (blockManager.GetHeight() == 0)
            {
                blockSynchronizer.StartFastSync();    
            }
            
            consensusManager.Start((long)blockManager.GetHeight() + 1);
            validatorStatusManager.Start(false);
```
This code is working for me on local testnet (start 4 nodes-validators, stop 1 of them,  clear its database,  start it again after 50 blocks are published)